app/src/common/shared/com/igalia/wolvic/ui/widgets/NavigationBarWidget.java

```diff
@@ -899,10 +882,6 @@ private void enterVRVideo(@VideoProjectionMenuWidget.VideoProjectionFlags int aP
         if (mFullScreenMedia != null && mFullScreenMedia.getWidth() > 0 && mFullScreenMedia.getHeight() > 0) {
             mediaWidth = (int) mFullScreenMedia.getWidth();
             mediaHeight = (int) mFullScreenMedia.getHeight();
-        } else if (needsYoutubeVideoWorkaround()) {
-            // Default to the size of a 4K video, which is our preferred video quality on YouTube.
-            mediaWidth = 3840;
```

Comment:
Reverting this as we are upgrading to Gecko 116 and that issue has already been fixed. https://bugzilla.mozilla.org/show_bug.cgi?id=1827583

And this can be the culprit for https://github.com/Igalia/wolvic/issues/923

Reply:
Are you sure the bug is fixed? We have applied that patch downstream in our Gecko fork but the workaround was still needed. Maybe that fix was not enough and required some other changes that we're getting now with 116.

I'll test that and get back to you.

Reply:
The problem were not the media controls, but some race conditions that prevented wolvic from showing the videos in fullscreen and/or immersive IIRC.

Anyway I've been testing this for a while and indeed it looks like we get the previous behaviour which was very nice as it saved some clicks for users.